owltorrent-common/src/main/java/edu/rice/owltorrent/common/util/Exceptions.java

```diff
@@ -14,4 +16,11 @@ public IllegalByteOffsets(String message) {
   }
 
   public static class ParsingTorrentFileFailedException extends Exception {}
+
+  @Getter
+  public static class BTException extends RuntimeException {
```

Comment:
Should this extend Runtime Exception or Exception/IOException? Note that RuntimeExceptions are all "sneakily thrown" and don't require people to catch it.